### PR TITLE
Credits page + Misc adjustments

### LINKED ISSUE #74: Responsive Fonts
We should probably work on responsive fonts for the page. I have read about some librarys that should work fine for it. I can look into this further and report my findings, i will probably also implement it for the clips page as well.
Is there anything that I have to look out for?

### LINKED ISSUE #92: Fixing size, gaps etc
Now that most of the pages is made, I think we need tidy up a lot of things. This includes adjusting font sizes #74 and adjusting margin and padding so that the gap is more uniform across component (current there are component with no gap or too much gap)

### PR DESCRIPTION
**Pull request by keijeizei**
- [x] Create basic credits layout
- [x] Finish Credits page (just waiting for the other messages from other projects)

Misc:
- [x] Add Coco's picture at the title
- [x] Make fonts with `em` units responsive using breakpoints in `globals.scss` Fixes #74 
- [x] Fix gap sizes Fixes #92 
- [x] Add /fanart in footer

### CODE REVIEW

File: packages/frontend/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -95,7 +95,12 @@ export default function Navigation({
         </div>
       </div>
 
-      {mode != Mode.FORM && <SakuraParticles />}
+      {mode != Mode.FORM && (
```
Comment (RedMaps) [inline packages/frontend/components/Navigation/Navigation.tsx]:
```suggestion
      {mode !== Mode.FORM && (
```

File: packages/frontend/pages/credits/index.tsx [hunk 2]
```diff
@@ -1,12 +1,96 @@
...
+        of your Hololive Fan Family. Remember, You will always remain our
+        24-Hour Kaichou!
+      </p>
+      <h2 className={styles.title}>Kaichou Project</h2>
+      <p className={styles.content}>
+        Dear Coco,
+        <br />
+        <br /> Thank you for all the smiles and laughter you&apos;ve given us
+        this past year and a half. We appreciate all the effort you put in to
+        innovate the world of vtubers and paving the way for HoloEN. We grateful
```
Comment (RedMaps) [inline packages/frontend/pages/credits/index.tsx]:
```suggestion
        innovate the world of vtubers and paving the way for HoloEN. We&apos;re grateful
```

**Review by RedMaps — CHANGES_REQUESTED** [review]
Job well done, some minor issues and I'd advocate for giving the credits page a big padding on the left and right so the text is easier to read and the visuals are improved, maybe around this much:
Screenshot 2021-07-01 at 11 21 50
Edit: I feel like the developers section should be a list from top to bottom... It kind of looks confusing to read in this horizontal way

**Comment by nyxordinal** [comment]
The page is already good, one comment if I may, could you increase the font size? 
I think it's a bit difficult for eyes to read the text in current font size

**Review by nyxordinal — APPROVED** [review]
Nice work !
I have no comments on this PR, if @RedMaps also has no comment then we can proceed to merge this PR

**Review by RedMaps — APPROVED** [review]
Looks good now!

**Pull request merged**